LargeBarrelAnalysisExtended/SignalFinder.cpp

```diff
@@ -22,83 +22,82 @@ using namespace std;
-  }
-  if (opts.count(fLeadTrailMaxTimeParamKey)) {
-    kSigChLeadTrailMaxTime = std::atof(opts.at(fLeadTrailMaxTimeParamKey).c_str());
-  }
-
-  fBarrelMap.buildMappings(getParamBank());
-  if (fSaveControlHistos) {
-    getStatistics().createHistogram(new TH1F("remainig_leading_sig_ch_per_thr", "Remainig Leading Signal Channels", 4, 0.5, 4.5));
-    getStatistics().createHistogram(new TH1F("remainig_trailing_sig_ch_per_thr", "Remainig Trailing Signal Channels", 4, 0.5, 4.5));
-    getStatistics().createHistogram(new TH1F("TOT_thr_1", "TOT on threshold 1 [ns]", 100, 20.0, 100.0));
-    getStatistics().createHistogram(new TH1F("TOT_thr_2", "TOT on threshold 2 [ns]", 100, 20.0, 100.0));
-    getStatistics().createHistogram(new TH1F("TOT_thr_3", "TOT on threshold 3 [ns]", 100, 20.0, 100.0));
-    getStatistics().createHistogram(new TH1F("TOT_thr_4", "TOT on threshold 4 [ns]", 100, 20.0, 100.0));
-  }
+	INFO("Signal finding started.");
+
+	if (opts.count(fEdgeMaxTimeParamKey)) {
+		kSigChEdgeMaxTime = std::atof(opts.at(fEdgeMaxTimeParamKey).c_str());
+	}
+
+	if (opts.count(fLeadTrailMaxTimeParamKey)) {
+		kSigChLeadTrailMaxTime = std::atof(opts.at(fLeadTrailMaxTimeParamKey).c_str());
+	}
+
+	if (fSaveControlHistos) {
+		getStatistics().createHistogram(
+			new TH1F("remainig_leading_sig_ch_per_thr",
+				"Remainig Leading Signal Channels",
+				4, 0.5, 4.5));
+		getStatistics().createHistogram(
+			new TH1F("remainig_trailing_sig_ch_per_thr",
+				"Remainig Trailing Signal Channels",
+				4, 0.5, 4.5));
+	}
 }
 
 //SignalFinder execution method
 void SignalFinder::exec()
 {
 
-  //getting the data from event in propriate format
-  if (auto timeWindow = dynamic_cast<const JPetTimeWindow* const>(getEvent())) {
+	//getting the data from event in apropriate format
+	if(auto timeWindow = dynamic_cast<const JPetTimeWindow* const>(getEvent())) {
+
+		//mapping method invocation
+		map<int, vector<JPetSigCh>> sigChsPMMap = SignalFinderTools::getSigChsPMMapById(timeWindow);
 
-    //mapping method invocation
-    map<int, vector<JPetSigCh>> sigChsPMMap = SignalFinderTools::getSigChsPMMapById(timeWindow);
-    //building signals method invocation
-    vector<JPetRawSignal> allSignals = SignalFinderTools::buildAllSignals(timeWindow->getIndex(), sigChsPMMap, kNumOfThresholds , getStatistics(), fSaveControlHistos,  kSigChEdgeMaxTime, kSigChLeadTrailMaxTime);
-    //saving method invocation
-    saveRawSignals(allSignals);
-  }
+		//building signals method invocation
+		vector<JPetRawSignal> allSignals = SignalFinderTools::buildAllSignals(
+							timeWindow->getIndex(),
+    							sigChsPMMap,
+    							kNumOfThresholds ,
+    							getStatistics(),
+    							fSaveControlHistos,
+    							kSigChEdgeMaxTime,
+    							kSigChLeadTrailMaxTime);
+
+		//saving method invocation
+		saveRawSignals(allSignals);
+
+	}
 }
 
 //SignalFinder finish method
 void SignalFinder::terminate()
 {
-  INFO("Signal finding ended.");
+	INFO("Signal finding ended.");
```

Comment:
I think you use tab -> 4 space  everywhre
We are using 2 spaces as a standard.
And astyle to set it.
It has not the highest  priority but it would avoid chanign the formatting back and forth